### ISSUE TITLE
A11y_VscodeWebviewUiToolkit_NumberField_ScreenReader: NVDA announces only "Clickable" for the increase/decrease spin button when navigating in browse mode using down arrow key

### ISSUE THREAD

**Issue by Shivamd1** [microsoft/vscode-webview-ui-toolkit#213]
### Actual:
NVDA announces only "Clickable" for the increase/decrease spin button when navigating in browse mode using down arrow key

Similar issue is observed with Jaws

### Expected:
NVDA should be announcing the proper name  for the increase/decrease spin button when navigating in browse mode using down arrow key. It should announce like "Increase" button

**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev3;#SH_VscodeWebviewUiToolkit_Aug2021;#Webview UI Toolkit for Visual Studio Code;#Win32;#FTP;#DesktopApp;#WCAG1.3.1;#NVDA;#Jaws;#STP;#AINotInScope;

### Environment Details:
Application Name: Vscode Webview Ui Toolkit
Windows Version: Win10
Screen Reader: NVDA(2021.1), Jaws(2021.2102.34)

### Repro Steps:

1. Launch VS Code
2. Open WebView UI Toolkit: All Component extension
3. Tab to "Number field: spin button and navigate to increase/decrease button in browse mode using down arrow key
4. Check the announcement

### User Impact:
Screen Reader users will not get the proper information about increase/decrease button

[“Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”
